sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -285,7 +329,18 @@ static void *_event_loop(void *args)
                 reply.content.value = -ENOTSUP;
                 msg_reply(&msg, &reply);
                 break;
-
+            case GNRC_IPV6_EXT_HANDLE_NEXT_HDR:
```

Comment:
Ok, but in this case please surround the case with a suitable `#ifdef MODULE_...`